src/OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper/Logger.cs

```diff
@@ -14,23 +14,23 @@
 // limitations under the License.
 // </copyright>
 
-using OpenTelemetry.AutoInstrumentation.DuckTyping;
+using OpenTelemetry.AutoInstrumentation.Logging;
 
-namespace OpenTelemetry.AutoInstrumentation.Instrumentations.GraphQL;
+namespace OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper;
 
-/// <summary>
-/// GraphQL.ErrorLocation interface for ducktyping
-/// </summary>
-[DuckCopy]
-internal struct ErrorLocationStruct
+internal static class Logger
 {
-    /// <summary>
-    /// Gets the line number of the document where the error occurred
-    /// </summary>
-    public int Line;
+    private static readonly ILogger Log = OtelLogging.GetLogger();
 
-    /// <summary>
-    /// Gets the column number of the document where the error occurred
-    /// </summary>
-    public int Column;
+    internal static void LogTrace(string message)
+    {
+        Log.Information(message);
+        BootstrapperEventSource.Log.Trace(message);
```

Comment:
Not directly related, but I think we should make the logging and EventSource usage consistent. `AutoInstrumentationEventSource` looks to be used in a different way. Additionally, probably we could have a single base class for all of our event sources.

I have also noticed that all event sources' `Trace` methods have `Level = EventLevel.Informational` (sic!). We should clean up these levels.

This comment should be addressed in a seperate PR. If you agree, you can create an issue out of this comment.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/2017